ledger/participant-integration-api/src/main/scala/platform/index/IndexServiceOwner.scala

```diff
@@ -167,12 +167,12 @@ final class IndexServiceOwner(
       dbSupport = dbSupport,
       eventsPageSize = config.eventsPageSize,
       eventsProcessingParallelism = config.eventsProcessingParallelism,
-      acsIdPageSize = config.acsIdPageSize,
-      acsIdPageBufferSize = config.acsIdPageBufferSize,
-      acsIdPageWorkingMemoryBytes = config.acsIdPageWorkingMemoryBytes,
-      acsIdFetchingParallelism = config.acsIdFetchingParallelism,
-      acsContractFetchingParallelism = config.acsContractFetchingParallelism,
-      acsGlobalParallelism = config.acsGlobalParallelism,
+      acsMaxNumberOfIdsPerIdPage = config.acsIdPageSize,
```

Comment:
Does this rename bring value, given that now the chain of names is broken and the same name cannot be followed from hocon all the way to the implementation?

Reply:
I see two issues:
1) being able to independently manage config keys names vs. corresponding names used in the Scala code,
2) having well and consistently named config keys and Scala names.




FYI I'm putting this PR on hold